src/components/SignIn/SignIn.styles.js

```diff
@@ -0,0 +1,16 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+export const useStyles = makeStyles((theme) => ({
+  form: () => ({
+    width: "30%",
+    marginLeft: "auto",
+    marginRight: "auto",
```

Comment:
You can create only one property margin with value '0 auto'

Reply:
Done